exchange/utils_test.go

```diff
@@ -135,6 +135,80 @@ func TestCleanOpenRTBRequestsCCPA(t *testing.T) {
 	}
 }
 
+func TestCleanOpenRTBRequestsSChain(t *testing.T) {
+	testCases := []struct {
+		description  string
+		inSourceExt  json.RawMessage
+		inExt        json.RawMessage
+		outSourceExt json.RawMessage
+		outExt       json.RawMessage
+	}{
+		{
+			description:  "Empty root ext and source ext",
+			inSourceExt:  json.RawMessage(``),
+			inExt:        json.RawMessage(``),
+			outSourceExt: json.RawMessage(``),
+			outExt: json.RawMessage(``),
+		},
+		{
+			description:  "No schains in root ext and empty source ext",
+			inSourceExt:  json.RawMessage(``),
+			inExt:        json.RawMessage(`{"prebid":{"schains":[]}}`),
+			outSourceExt: json.RawMessage(``),
+			outExt:       json.RawMessage(`{"prebid":{}}`),
```

Comment:
I'm not sure if the expected behavior should be to return an empty `prebid` or if the key should be deleted. I think deleting the key makes more sense but I didn't go down that path because of performance concerns when performing a deep equal comparison using the reflect package to check for an empty struct prior to marshaling. I also don't think you can compare the struct that maps to `prebid` to a zero value struct since it has nested structs. I'm looking for some feedback here.